oauth2client/contrib/appengine.py

```diff
@@ -477,15 +469,15 @@ def _parse_state_value(state, user):
         user: google.appengine.api.users.User, The current user.
 
     Raises:
-        InvalidXsrfTokenError: if the XSRF token is invalid.
+        ValueError: if the XSRF token is invalid.
```

Comment:
One difference is that `flask_util` stores the server XSRF token in a session, whereas `appengine` regenerates the token during validation.